Latest header does not compile on MSVC due to const expressions

The following line doesn't compile on MSVC anymore, it seems:

https://github.com/webgpu-native/webgpu-headers/blob/2b5974750be7b2545d842ecec02e585022081952/webgpu.h#L931

```
E:\Projects\wgpu-native\ffi\webgpu-headers\webgpu.h(931,33): error C2099: initializer is not a constant [E:\Projects\wgpu-native\examples\build\framework\framework.vcxproj]
```

According to this very authorative source (stack overflow), `const int` declarations are not const themselves in C: https://stackoverflow.com/questions/6131455/compile-error-c2099-initializer-is-not-a-constant, so that's probably why this doesn't work.

Found this while trying to update wgpu-native to the latest headers.

--- a/gen/gen.go
+++ b/gen/gen.go
@@ -270,40 +270,64 @@ func (g *Generator) EnumValue(prefix string, e Enum, entryIndex int) (string, er
 	return fmt.Sprintf("%s%.4X", prefix, entryValue), nil
 }
 
+func bitflagEntryValue(entry BitflagEntry, entryIndex int) (uint64, error) {
+	if entry.Value == "" {
+		value := uint64(math.Pow(2, float64(entryIndex-1)))
+		return value, nil
+	} else {
+		var num string
+		var base int
+		if strings.HasPrefix(entry.Value, "0x") {
+			base = 16
+			num = strings.TrimPrefix(entry.Value, "0x")
+		} else {
+			base = 10
+			num = entry.Value
+		}
+		return strconv.ParseUint(num, base, 64)
+	}
+}
+
 func (g *Generator) BitflagValue(b Bitflag, entryIndex int) (string, error) {
 	entry := b.Entries[entryIndex]
-	var entryValue string
+
+	var value uint64
+	var entryComment string
 	if len(entry.ValueCombination) > 0 {
+		if entry.Value != "" {
+			return "", fmt.Errorf("BitflagValue: found conflicting 'value' and 'value_combination' in '%s'", b.Name)
+		}
+		entryComment += " /* "
 		for valueIndex, v := range entry.ValueCombination {
-			entryValue += "WGPU" + PascalCase(b.Name) + "_" + PascalCase(v)
+			// find the value by searching in b, bitwise-OR it into the result
+			for searchIndex, search := range b.Entries {
+				if search.Name == v {
+					searchValue, err := bitflagEntryValue(search, searchIndex)
+					if err != nil {
+						return "", nil
+					}
+					value |= searchValue
+					break
+				}
+			}
+			// construct comment
+			entryComment += PascalCase(v)
 			if g.ExtSuffix != "" {
-				entryValue += "_" + g.ExtSuffix
+				entryComment += "_" + g.ExtSuffix
 			}
 			if valueIndex != len(entry.ValueCombination)-1 {
-				entryValue += " | "
+				entryComment += " | "
 			}
 		}
+		entryComment += " */"
 	} else {
-		if entry.Value == "" {
-			value := uint64(math.Pow(2, float64(entryIndex-1)))
-			entryValue = fmt.Sprintf("0x%.16X", value)
-		} else {
-			var num string
-			var base int
-			if strings.HasPrefix(entry.Value, "0x") {
-				base = 16
-				num = strings.TrimPrefix(entry.Value, "0x")
-			} else {
-				base = 10
-				num = entry.Value
-			}
-			value, err := strconv.ParseUint(num, base, 64)
-			if err != nil {
-				return "", err
-			}
-			entryValue = fmt.Sprintf("0x%.8X", value)
+		var err error
+		value, err = bitflagEntryValue(entry, entryIndex)
+		if err != nil {
+			return "", nil
 		}
 	}
+	entryValue := fmt.Sprintf("0x%.16X", value) + entryComment
 	return entryValue, nil
 }
 

--- a/webgpu.h
+++ b/webgpu.h
@@ -928,7 +928,7 @@ static const WGPUColorWriteMask WGPUColorWriteMask_Red = 0x0000000000000001;
 static const WGPUColorWriteMask WGPUColorWriteMask_Green = 0x0000000000000002;
 static const WGPUColorWriteMask WGPUColorWriteMask_Blue = 0x0000000000000004;
 static const WGPUColorWriteMask WGPUColorWriteMask_Alpha = 0x0000000000000008;
-static const WGPUColorWriteMask WGPUColorWriteMask_All = WGPUColorWriteMask_None | WGPUColorWriteMask_Red | WGPUColorWriteMask_Green | WGPUColorWriteMask_Blue | WGPUColorWriteMask_Alpha;
+static const WGPUColorWriteMask WGPUColorWriteMask_All = 0x000000000000000F /* Red | Green | Blue | Alpha */;
 
 typedef WGPUFlags WGPUMapMode;
 static const WGPUMapMode WGPUMapMode_None = 0x0000000000000000;

--- a/webgpu.yml
+++ b/webgpu.yml
@@ -1410,7 +1410,6 @@ bitflags:
           TODO
       - name: all
         value_combination:
-          - none
           - red
           - green
           - blue

